codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/Encoder.java

```diff
@@ -304,9 +305,7 @@ private HeaderEntry getEntry(CharSequence name, CharSequence value) {
         int h = hash(name);
         int i = index(h);
         for (HeaderEntry e = headerFields[i]; e != null; e = e.next) {
-            if (e.hash == h &&
-                    HpackUtil.equals(name, e.name) &&
-                    HpackUtil.equals(value, e.value)) {
+            if (e.hash == h && (equalsConstantTime(name, e.name) & equalsConstantTime(value, e.value)) != 0) {
```

Comment:
I asked internally how "constant" a constant function needs to be, and so far the answer has been "nothing is constant and will basically be impossible with an optimizing compiler / processor".

This is best effort, and copying behavior of other security reviewed code feels like said effort.